backend-services/voting-app/src/main/java/org/cardano/foundation/voting/domain/entity/Vote.java

```diff
@@ -60,7 +60,7 @@ public class Vote extends AbstractTimestampEntity {
     private String signature;
 
     @Column(name = "payload", nullable = false, columnDefinition = "text", length = 2048)
-    @Nullable
+    @Nullable // TODO remove nullable since payload is now always required
```

Comment:
in future releases, don't wanna do db migration now, too risky and not necessary.